samples/snippets/src/main/java/VerifyGoogleIdToken.java

```diff
@@ -26,9 +26,8 @@ public static void main(String[] args) {
     // The Google ID token to verify.
     String idToken = "id-token";
 
-    // The service name for which the id token is requested. Service name refers to the
-    // logical identifier of an API service, such as "pubsub.googleapis.com".
-    String targetAudience = "pubsub.googleapis.com";
+    // The service name for which the id token is requested.
```

Comment:
This is interesting. If you are validating an ID token, you should be checking what is in the token against what you expect, right? So wouldn't this be something like 'The expected target, for example, "https://example.com"'?

Reply:
We are not doing a manual check here. Instead we let the API verify by providing the details that we used during creating the ID token.
Hence, I would prefer to leave it as such.